cognite/powerops/clients/cogshop/data_classes/_core.py

```diff
@@ -53,7 +53,7 @@ class DomainModel(DomainModelCore):
     def from_node(cls, node: dm.Node) -> T_TypeNode:
         data = node.dump(camel_case=False)
 
-        return cls(**{**data, **unpack_properties(node.properties)})
+        return cls(**data, **unpack_properties(node.properties))
```

Comment:
Just to verify... This change with the dict "re-packing" was to avoid an error, has this been fixed in pygen? 
```
po_client.dm.model_templates.retrieve(external_id=known_ext_id)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[3], line 1
----> 1 c.dm.model_templates.retrieve(external_id="ModelTemplate_Fornebu")

File /.../power-ops-sdk/cognite/powerops/client/dm_client/_api/model_templates.py:122, in ModelTemplatesAPI.retrieve(self, external_id)
    120 def retrieve(self, external_id: str | Sequence[str]) -> ModelTemplate | ModelTemplateList:
    121     if isinstance(external_id, str):
--> 122         model_template = self._retrieve(("cogShop", external_id))
    123         model_edges = self.model.retrieve(external_id)
    124         base_mapping_edges = self.base_mappings.retrieve(external_id)

File /.../power-ops-sdk/cognite/powerops/client/dm_client/_api/_core.py:53, in TypeAPI._retrieve(self, nodes)
     51 if is_multiple:
     52     return self.class_list([self.class_type.from_node(node) for node in instances.nodes])
---> 53 return self.class_type.from_node(instances.nodes[0])

File /.../power-ops-sdk/cognite/powerops/client/dm_client/data_classes/_core.py:56, in DomainModel.from_node(cls, node)
     52 @classmethod
     53 def from_node(cls, node: dm.Node) -> T_TypeNode:
     54     data = node.dump(camel_case=False)
---> 56     return cls(**data, **unpack_properties(node.properties))

TypeError: cognite.powerops.client.dm_client.data_classes._model_templates.ModelTemplate() got multiple values for keyword argument 'version'
```

Reply:
Thanks for that one. I know `pygen` is running into an issue if you have a field named `version` as we do. I am waiting for an answer from product whether `version` will be a reserved keyword in DM.